odkx-src/build-scripts.rst

```diff
@@ -148,7 +148,27 @@ To run the UI tests:
 
       ./gradlew clean testSnapshotBasicDebug connectedSnapshotBasicDebugAndroidTest
 
+Build Variants
+~~~~~~~~~~~~~~~~~~
+
+you can configure build variants to create different versions of your app from a single project, and how to properly manage your dependencies and signing configurations. Each build variant represents a different version of your app that You can configure a dependency for a specific build variant. For more information refer to https://developer.android.com/studio/build/build-variants
+
+The Build variant corresponding to the source branches:
+
+1. **development**:  **snapshotBasicRelease** is one of the build variant corresponds to development branch in Android studio. All the required changes and forks are done from this branch in which the new development, upgrades and features should be contributed and tested.
```

Comment:
It's not limit to Android Studio, it includes the command line tool Gradle. I would drop the word Android studio here and just have a section below telling people how to change the build variant in a sub-section about Android Studio after this list.

Reply:
okay sure I got it. But I have written the steps to change the build variant after this list.